rcv.iml

```diff
@@ -1,2 +1,370 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<module external.linked.project.id="rcv" external.linked.project.path="$MODULE_DIR$" external.root.project.path="$MODULE_DIR$" external.system.id="GRADLE" external.system.module.group="" external.system.module.version="unspecified" type="JAVA_MODULE" version="4" />
\ No newline at end of file
+<module relativePaths="true" type="JAVA_MODULE" version="4">
```

Comment:
What are the implications of this change? Is this just an IntelliJ file that sets the project up in a certain way?

Reply:
Yeah, I believe this is just Intellij configuration.

Reply:
I see. The problem is it's referencing your local environment (see "tamird" appearing quite a bit in there). I'd say either figure out a way to make this more general / relative paths, or revert these changes.